content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  apps/reference/app-settings
+  apps/guides/performance/replication/#depth
+  core/overview/transitions 
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including;
+
+- Setting user roles and permissions
+- Enabling and disabling transitions
+- Configuring contact hierarchy and configuring replication.
+
+App settings allow you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
+
+## Brief Overview of Key Concepts
+
+The settings which control CHT apps are defined in the *app_settings.json* file, and stored in the settings doc in the database.
+
+*Permissions* are settings that control access to specific app features and functionality.
```

Comment:
add link to Permissions: {{< relref "apps/reference/app-settings/user-permissions" >}}

This is just a placeholder page currently, but should be updated in the near future